src/shared/ui/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react"
+import './SearchForm.css'
+
+export const SearchForm = ({ initialValue = "", onSearch }) => {
+  const [searchText, setSearchText] = useState(initialValue);
+
+  const handleInputChange = (event) => {
+    setSearchText(event.target.value);
+  };
+
+  const handleSearchClick = () => {
+    onSearch(searchText);
+  };
+
+  const handleKeyPress=(event)=>{
+    if (event.key === "Enter") {
+      onSearch(searchText);
+    }
+  }
+
+  return (
+    <form class="searchform inputelement">
```

Comment:
best practice for css classes is` kebab-case`
```suggestion
    <form class="search-form input-element">
```

Reply:
fixed